ansible/roles/update/templates/50unattended-upgrades.j2

```diff
@@ -0,0 +1,23 @@
+// Managed by Ansible
+
+// Controls which packages are upgraded
+Unattended-Upgrade::Origins-Pattern {
+        "origin=Debian,codename=${distro_codename},label=Debian";
+        "origin=Debian,codename=${distro_codename},label=Debian-Security";
+};
+
+{% if unattended_minimal_steps %}
+Unattended-Upgrade::MinimalSteps "true";
+{% endif %}
+
+{% if unattended_mail %}
+Unattended-Upgrade::Mail "{{ admin_email }}"; //TODO
```

Comment:
I was looking into this the other day and I remember seeing that we need to have the email configured on the machine. Have you checked / tested this?

Reply:
I forgot about this TODO, I updated the role to add support for a SMTP account that sends out email notifications.